src/functionalTest/resources/features/F-201 - Assign Access within Organisation/F-201 - Assign Access within Organisation.feature

```diff
@@ -13,7 +13,7 @@ Feature: F-201: Assign Access within Organisation
     Given a user [S1 - a solicitor, to create a case under their organisation and share it with a fellow solicitor in the same organisation],
       And a user [S2 - another solicitor in the same organisation, with whom S1 will share a case with an assignment within organisation],
       And a case [C1, which S1 has just] created as in [F-201_Prerequisite_Case_Creation_C1],
-      And a wait time of [5] seconds [to allow for the case just created to appear in search results],
+      And a wait time of [8] seconds [to allow for the case just created to appear in search results],
```

Comment:
8 Seconds is too much, but I understand it is for safety.
Why would cases occasionally take so long to appear in ES results?
Do we have a ticket to fix the actual problem?

Reply:
The **5 threads & 8 second wait is a compromise to avoid 'Case not found' Elastic Search issues in local development** when running FTA tests locally against MCA running at console or IDE and CCD in docker (4 CPU, 10GB + 2 GB swap, with active compose files: `backend sidam sidam-local sidam-local-ccd elasticsearch frontend`).

I tried the same 10 threads as the other repos and with the original 5 seconds wait however as ElasticSearch is used throughout MCA's APIs local failures were too frequent.   So after trying many different combinations.  5+8 was the quickest complete time to successfully run current FTAs (201, 202, 203 + 206) more often than hitting a 'Case not found' failure, **although it still often does**.

Reply:
@mattnayler  - I understand the current situation, but could you please raise a new ticket to remediate the issue in local setups? The team won't fix it, though, as priorities will not allow.

Reply:
@MSancaktutar I have now added a [comment to ACA-99](https://tools.hmcts.net/jira/browse/ACA-99?focusedCommentId=898484#comment-898484)  _"Intermittent MCA FTA test results running locally on ccd-docker stack"_ referring to the MCA thread count and wait time being different in MCA: and occasional faults still occuring.